content/verein/board.fr.md

```diff
@@ -0,0 +1,25 @@
+{
+    "title": "Le comité directeur",
+    "slug": "comite-directeur",
+    "aliases": ["vorstand", "vorsitzende", "vorsitzender", "chair"],
+    "type": "page"
+}
+
+Toutes les affaires importantes de l'association sont menées par le comité directeur. Conformément à notre [constitution]({{< ref "verein/constitution" >}}#7-the board), le comité directeur est composé de deux présidents paritaires, qui représentent conjointement l'association, et d'un nombre arbitraire de conseillers, qui contrôlent et soutiennent le comité directeur dans son travail. Le comité actuel a été élu lors de l'assemblée constitutive du 14 juillet 2018 et se compose de :
+
+* Benjamin Altpeter, Président associé ([benni@datenanfragen.de](mailto:benni@datenanfragen.de), [Clé PGP](/pgp/AF1381BD.asc) `9F99 3663 0B82 D3C7 DFC9  23F6 EB5C F074 AF13 81BD`)
+* Lorenz Sieben, Président associé ([lorenz@datenanfragen.de](mailto:lorenz@datenanfragen.de), [Clé PGP](/pgp/4C44A6EF.asc) `5B50 D7A0 7585 26D4 55AA  F305 DFAF 12BB 4C44 A6EF`)
+
+Il n'y a pour le moment aucun conseiller au sein du comité directeur.
+
+Tu peux contacter le comité directeur de différentes façons :
+
+* par e-mail à l'adresse [vorstand@datenanfragen.de](mailto:vorstand@datenanfragen.de) ([Clé PGP](/pgp/3E2B296B.asc) `2E72 EA5B DDE3 1730 58D7  F87D A0C1 C012 3E2B 296B`)
+* par téléphone au +49 531 209299 35 (pas d'horaires de bureau)  
```

Comment:
I was not sure about this. What do you mean exactly by "(no office hours)".

Reply:
We don't have any business hours when you can definitely reach us by phone. That phone number just redirects to Lorenz' mobile. He'll answer if possible but he might also be busy.